gcp/redis_location_list.go

```diff
@@ -0,0 +1,56 @@
+package gcp
+
+import (
+	"context"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"google.golang.org/api/iterator"
+	"google.golang.org/genproto/googleapis/cloud/location"
+)
+
+const matrixKeyRedisLocation = "redis-location"
+
+func BuildRedisLocationList(ctx context.Context, d *plugin.QueryData) []map[string]interface{} {
```

Comment:
@madhushreeray30, could you please add a comment pointing to this function requirements?